### ISSUE TITLE
Update command for ffmpeg configuration

### ISSUE THREAD

**Issue by nicklasfrahm** [jaredpetersen/raspi-live#20]
Quick fix to the `README.md`. We should probably update it to use a different configuration command as the Raspberry Pi 3B+ has an `armhf` CPU architecture, not `armel`.

The fix could simply be: `sudo ./configure --arch=$(dpkg --print-architecture) --target-os=linux --enable-gpl --enabl
e-omx --enable-omx-rpi --enable-nonfree`

Will submit a PR.

**Comment by jaredpetersen** [comment]
Good catch!

I don't have a free raspberry pi to test this on and I'm not familiar with dpkg. Have you been able to test that `dpkg --print-architecture` works for both the new and old raspberry pis? I'd love to be able to just swap out part of the installation script but I want to make sure that it's compatible with all raspberry pis.

It may be better to just split up the installation commands like what was done with step 2 to deal with the varying number of CPU cores. Maybe something like:

1. Dowload FFmpeg via:
```
sudo apt-get install libomxil-bellagio-dev
wget -O ffmpeg.tar.bz2 https://ffmpeg.org/releases/ffmpeg-snapshot-git.tar.bz2
tar xvjf ffmpeg.tar.bz2
```
2. Find out what kind of CPU architecture your raspberry pi uses. Raspberry Pi 3B+ and later uses `armfh` and everything earlier uses `armel`. Then configure FFmpeg via the following command, replacing CPUARCHITECTUREGOESHERE with your raspberry pi's CPU architecture:
```
sudo ./configure --arch=CPUARCHITECTUREGOESHERE --target-os=linux --enable-gpl --enable-omx --enable-omx-rpi --enable-nonfree
```
3. Find out how many CPU cores your raspberry pi has. If your raspberry pi is single core, just run `sudo make`. If your raspberry pi has a quad-core CPU, run `sudo make -j4` to build FFmpeg.

**Comment by nicklasfrahm** [comment]
@jaredpetersen The main reason why I used a command was that I did not remember the exact CPU architecture for each Raspberry Pi by heart. I only caught this, because I roughly remembered that the CPU architecture was `armhf` for the newer ones. Using a script would also make it less susceptible to human error, because the CPU info should always be right. I should have a RPi 3 lying around. I will check what its CPU architecture is compared to the RPi 3B+.

**Comment by jaredpetersen** [comment]
Sounds good to me! If there's a good way to script it that works across all raspberry pi models, I'm all for it 👍

**Comment by nicklasfrahm** [comment]
@jaredpetersen Should I create a small bash script that compiles and installs `ffmpeg`, because then we can add the good ol' `curl | bash` to the `README.md`?

**Comment by jaredpetersen** [comment]
That would definitely make the installation process a lot easier and remove ambiguity around figuring out what kind of CPU your raspberry pi has (architecture, number of cores, etc.). However, a lot of people like to see exactly what an installation script that requires `sudo` permissions is doing and rightfully don't trust running "random" shell scripts they downloaded off the internet.

Granted, they could look at the script first but I think it's more transparent to have the user run through the installation steps line by line, particularly since there aren't that many.

**Comment by nicklasfrahm** [comment]
@jaredpetersen Totally, get your point. What about the #21 then for now and I add the installation script as optional step afterwards? Do you need anything more from my side to approve it?

As for the script, I kinda like how [helm](https://helm.sh) does it:
> Yes, you can `curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | bash` that if you want to live on the edge.

**Comment by nicklasfrahm** [comment]
Would you mind if I also throw in an editor config to prevent problems with whitespaces in PRs?

**Comment by jaredpetersen** [comment]
Resolved by #21
